packages/mongo-nav/src/mongo-select/Trigger.tsx

```diff
@@ -67,25 +72,44 @@ export function OrganizationTrigger({
   className,
   urls,
   isActive,
+  disabled = false,
   ...rest
 }: OrganizationTriggerProps) {
   return (
     <div className={cx(orgTriggerContainer, className)}>
-      <button {...rest} className={buttonContainer}>
-        <Icon size="small" glyph="Building" />
-        <span className={selectedStyle}>{placeholder}</span>
-        <Icon size="small" glyph="CaretDown" />
-      </button>
-      <a
-        href={urls.mongoSelect.orgSettings}
-        className={cx(anchorStyle, border)}
-        aria-label="settings"
+      <button
+        {...rest}
+        className={buttonContainer}
+        data-testid="org-trigger"
+        disabled={disabled}
       >
         <Icon
-          glyph={'Settings'}
-          fill={isActive ? uiColors.green.base : uiColors.gray.base}
+          className={cx({ [disabledStyle]: disabled })}
+          size="small"
+          glyph="Building"
         />
-      </a>
+        <span className={cx(selectedStyle, { [disabledStyle]: disabled })}>
+          {disabled ? 'All Organizations' : placeholder}
```

Comment:
I usually would be against this for a more generalized component, but since this is so specific, we can hard-code the placeholder value here rather than accepting it as a prop.